frontend/src/toolbar/button/Circle.tsx

```diff
@@ -7,7 +7,7 @@ interface CircleProps extends React.PropsWithoutRef<JSX.IntrinsicElements['div']
     extensionPercentage?: number
     distance?: number
     rotation?: number
-    content?: string | JSX.Element
```

Comment:
Why this change?

Reply:
It was throwing an error previously because the `JSX.IntrinsicElements['div']` type from which it extends also has a `content` prop. Just reverted it and eslint seems happy so will try without it again.